static/styles/widgets.css

```diff
@@ -167,3 +175,20 @@ button {
 .current-user-vote {
     background-color: hsla(156, 10%, 90%, 0.9);
 }
+
+.add-task-wrapper {
+    display: inline;
+    position: relative;
+    z-index: 1;
```

Comment:
Why is `z-index` needed here?

Reply:
When the button becomes disabled the z-index is set to -1. When testing it was evident that the wrapper needed to be strictly "on top" of the button in order for the tippy tooltip to be triggered ([this](https://user-images.githubusercontent.com/36671046/143785928-c6c309f9-8cec-49c0-8b4b-fe8bd8769ad1.mp4) is without z-index). This is also why the additional grayed out styling of the disabled button is determined by the hovering of the wrapper because it's now "on top".